config-jetson.ini

```diff
@@ -1,41 +1,39 @@
 [App]
-VideoPath: /repo/data/gard1-4.mp4
-Host: 0.0.0.0
```

Comment:
@emma-w-dev Please check if your test needs to be modified. These host and port settings for all platforms are now irrelevant and I have removed them from the related config files.